### ISSUE TITLE
Launch Infura analogue for Ethereum CLassic

### ISSUE THREAD

**Issue by Dexaran** [Dexaran/ICO#7]
https://infura.io/

**Comment by yograterol** [comment]
Make from Scratch or contribute to add the support to ETC?

**Comment by Dexaran** [comment]
I'm planning to get in touch with ConsenSys. If they express a desire to support the ETC, then the problem is solved.
If they will not support ETC, then I propose to use ConsenSys code base to build a similar service for ETC.

**Comment by ProphetDaniel** [comment]
I created an API endpoint yesterday with whisper protocol enabled, still synchronizing though. https://github.com/inzhoop-co/LETH/issues/27#

**Comment by yograterol** [comment]
If you want, I launch a ETC Node -> https://rpcgeth.viperid.online/

**Comment by Dexaran** [comment]
Yeah, it seems that there are plenty of free ETC open nodes and anyone can use them as API endpoints. I think that it's not the most necessary thing right now. 

Perhaps, when the ETC becomes more popular, we will need a service to run API endpoints.

**Comment by TomaszWaszczyk** [comment]
Anyone knows what is attitude ConsenSys on ETC?

**Comment by xiaods** [comment]
Anyone knows what is attitude ConsenSys on ETC?

**Comment by xiaods** [comment]
I have already found a solution:  https://www.ethercluster.com/
